### DISCUSSION TITLE
What is BAUET Result System?

### DISCUSSION THREAD

**Discussion by sourovkumarpaul14** [mdarikrayhan/BAUET_Result_System#3] [Q&A] [answered]
I would like to know what do you mean by BAUET Result System?

**Comment by mdarikrayhan** [comment, marked as answer]
Bauet result system is a website where the result of the students will be published.

**Reply by sourovkumarpaul14** [reply]
Thank for letting me know.

**Reply by mdarikrayhan** [reply]
You are welcome.

**Reply by mimfathma** [reply]
Thanks
